documentation/book/proc-creating-new-image-from-base.adoc

```diff
@@ -3,17 +3,22 @@
 // assembly-using-kafka-connect-with-plugins.adoc
 
 [id='creating-new-image-from-base-{context}']
-= Create a new image based on our base image
+= Creating a Docker image from the Kafka Connect base image
 
-{ProductName} provides its own container image for running Kafka Connect, which can be found on {DockerRepository} as
-`{DockerKafkaConnect}`.
-The `{DockerKafkaConnect}` image is configured to automatically load all plugins or connectors that are present in the `/opt/kafka/plugins` directory during startup.
-This image can be used as a base image for building a new custom image with additional plugins.
+A container image for running Kafka Connect using {ProductName} is available on {DockerRepository} as `{DockerKafkaConnect}`. You can use this as a base image when creating your own custom image with additional plug-ins. 
 
-The following procedure describes the process for creating such a custom image and adding the connector plugins to the the `/opt/kafka/plugins` directory.
+.Prerequisites
+
+ifdef::Kubernetes[]
+* xref:deploying-cluster-operator-kubernetes-{context}[Deploying the Cluster Operator to {KubernetesName}]
+endif::Kubernetes[]
+
+* xref:deploying-cluster-operator-openshift-{context}[Deploying the Cluster Operator to {OpenShiftName}]
```

Comment:
I believe you want to replace {context} here with whatever the context was on the page you're linking to